packages/x-charts/src/BarChart/BarLabel/getBarLabel.ts

```diff
@@ -0,0 +1,20 @@
+import { SeriesId } from '../../models/seriesType/common';
+import { BarLabelFunction } from './types';
+
+export const getBarLabel = (options: {
+  barLabel: 'value' | BarLabelFunction;
+  value: number | null;
+  dataIndex: number;
+  seriesId: SeriesId;
+  height: number;
+  width: number;
+}): string | null | undefined => {
+  const { barLabel, value, dataIndex, seriesId, height, width } = options;
+
+  if (barLabel === 'value') {
+    // We don't want to show the label if the value is 0
+    return value ? value?.toString() : null;
+  }
```

Comment:
What about using the `valueFormatter` here?

Reply:
I thought about it, but also thought it would be a bit odd. Like, the value is the raw value, you can run a formatter yourself if needed.

Another option would be to add `formattedValue` as one of the options of `barLabel`, and run the formatter ourselves.

But I don't think the `'value'` option should return the formatted value.

Do you think we should add a `formattedValue` option?

Reply:
We can wait to see if a user needs it